README.md

```diff
@@ -16,87 +16,208 @@ Author: **Anna Ivanova**
+                        or .txt file name which includes multiple SRA Study identifiers
+
+options:
+  -h, --help            show this help message and exit
+  -L {debug,info,warning,error}, --log {debug,info,warning,error}
+                        Logging level
+  -O OUT, --out OUT     Output directory
+  -M METHOD, --method METHOD
+                        Choose different type of methods that should be used for data retrieval:
+                        Aspera (a), FTP (f), fasterq_dump (q). By default it is fasterq_dump (q)
+  -c CORES, --cores CORES
+                        Number of CPU cores to utilise (for subcommands that support parallel execution)
+```
+
+This help message is also accessible via `python3 -m fastqheat --help`.
+
 ## Compatibility
 
-FastqHeat needs Python 3.7 or newer.
+FastqHeat is being developed and tested under Python 3.9.x.
 
 ## Installation
 
-Clone the project from GitHub or download it as an archive, then install it directly
-with `pip` (consider creating a fresh [virtual environment](https://docs.python.org/3/library/venv.html#creating-virtual-environments) for that). Example:
+ 1. [Make sure you have installed a supported version of Python](https://www.python.org/downloads/).
+
+ 2. Clone this project from GitHub or download it as an archive.
+
+ 3. **Optional, but recommended:** create and activate a fresh
+ [virtual environment](https://docs.python.org/3/library/venv.html#creating-virtual-environments).
+
+ 4. Install it directly with `pip`.
+
+Full example for Linux systems:
 
 ```bash
-~$ git clone git@github.com:quantori/FastqHeat.git
-~$ pip install FastqHeat/
+$ git clone git@github.com:quantori/FastqHeat.git
+$ python3 -m venv env
+$ . env/bin/activate
+$ pip install FastqHeat/
 ```
 
-Depending on the method you choose for downloading data, you may have to install additional
-command-line utilities, as explained below.
+This will install the project and its external Python dependencies (currently the only external Python
+dependency is `requests`). Depending on the method you choose for downloading data, you may have to install
+additional command-line utilities, as explained in the section below.
 
 ## Supported methods
 
 ### Fasterq-dump
 
 Requires `fasterq-dump` executable installed and added to `PATH`. Consult the
-[SRA Toolkit documentation](https://github.com/ncbi/sra-tools/wiki/HowTo:-Binary-Installation)
-for detailed instructions. After downloading files, this program will compress them with
+[official SRA Toolkit documentation](https://github.com/ncbi/sra-tools/wiki/HowTo:-Binary-Installation)
+for detailed instructions. After downloading files, FastqHeat will compress them with
 [`pigz`](https://github.com/madler/pigz) (can be installed with `apt` on Debian-based systems).
 
+Both `fasterq-dump` and `pigz` support parallel execution and it's enabled by default. The `-c`/`--cores`
+argument (see [CLI usage](#cli-usage)) controls exactly how many threads those programs will spawn.
```

Comment:
@dakq I may be wrong, but usually 'those' and 'that' are used instead of 'these' and 'this' in past tenses 

https://www.britannica.com/dictionary/eb/qa/This-these-that-and-those